doc/incentives/simulation-and-test-scenarios.md

```diff
@@ -1,87 +1,75 @@
+
+Also, some of the tests will need the presence of a simulated blockchain. The network testing track is already setting up the framework which will allow such testing by providing infrastructure accordingly (e.g. Ganache as a blockchain backend).
 
 We can thus roughly distinguish between:
 * Simulations
 * Integration tests
 * Experimentation tests
 
-This document aims at collecting the specification for the set of needed testing.
+This document aims to collect the specification for the set of needed testing.
 
 ## Simulations
-Simulations run as part of the go unit testing suite. 
-They spawn a configurable number of nodes in a single process and then can run
-individual interactions. They are mostly useful to run tests which should verify that
-a change in the code will not break integrity or correctness of existing code and which should
-run on every change of the code. They are not useful much for experimentation,
-nor for integration, as they should be focused on running fast to not extend too much
-the complete go unit suite. 
+Simulations run as part of the go unit testing suite. They spawn a configurable number of nodes in a single process and then can run individual interactions. They are mostly useful to verify that a change in the code does not break the integrity or correctness of the system, and thus should be run every time the code changes. They are not of much use for experimentation nor for integration, as they should be focused on running quickly so as not to significantly extend the running time of the complete test suite.
 
 ### Accounting correctness
-This is a test well suited for a simulation. At no point after a change the correctness of accounting should be broken.
-**Description**: Launch x nodes, upload a file, sync, then download from different nodes, and check balances.
-**Expectation**: All balances should be symmetric with each other (`balanceA == -balanceB && balanceB == -balanceA`)
-**Restriction**: Keep balances below threshold so to not emit cheques 
+This is a test well suited for a simulation. At no point after a change should the correctness of accounting be broken.
+
+**Description**: Launch _x_ nodes, upload a file, sync, then download from different nodes, and check balances.  
+**Expectation**: All balances should be symmetric with each other (`balanceA == -balanceB && balanceB == -balanceA`) and at least _some_ of them should be non-zero.  
+**Restriction**: All nodes should have SWAP enabled and a valid chequebook smart contract deployed. All thresholds should be equal and all nodes must be on the same network. Balances should be kept below the payment threshold so no cheques are emitted.  
```

Comment:
This restriction only applies for upload integration tests as mentioned above, when we have a properly deployed blockchain (ganache) and can deploy chequebooks.